pkg/admission/cluster_webhook.go

```diff
@@ -160,3 +163,12 @@ func ValidateDeleteCluster(ctx context.Context, _ client.Client, obj runtime.Obj
 	logger.Error(err, "deletion request denied", "cluster", cluster.GetName())
 	return admission.Warnings{"deletion is not allowed"}, err
 }
+
+func validateTokenValidity(cluster *greenhousev1alpha1.Cluster) (admission.Warnings, error) {
+	if cluster.Spec.MaxTokenValidity.Duration > greenhousev1alpha1.MaxTokenValidity {
+		err := apierrors.NewBadRequest("token validity is too long")
+		return admission.Warnings{"token validity too long"}, err
+	}
```

Comment:
I would probably overwrite the `MaxTokenValidity` to the default `72 hours` if the value is bigger, just to not block the user to onboard the cluster. Also somehow present the warning and explain that we did overwrite his value ( either UI and or via `greenhousectl`
@IvoGoman wdyt ?

Reply:
Since validation is needed here (I forgot before suggesting default) then we need to make `MaxTokenValidity`as an `int32` instead and then we can do the following annotations on top of it -

```go
// ClusterSpec defines the desired state of the Cluster.
type ClusterSpec struct {
	// AccessMode configures how the cluster is accessed from the Greenhouse operator.
	AccessMode ClusterAccessMode `json:"accessMode"`

	// MaxTokenValidity specifies the maximum duration for which a token remains valid
	// +kubebuilder:default:=72
	// +kubebuilder:validation:Minimum=24
	// +kubebuilder:validation:Maximum=72
	MaxTokenValidity *int32 `json:"maxTokenValidity,omitempty"`
        ...
}
```

Reply:
Done